data/txHandler_test.go

```diff
@@ -980,6 +987,7 @@ func TestTxHandlerProcessIncomingCacheBacklogDrop(t *testing.T) {
 
 func TestTxHandlerProcessIncomingCacheTxPoolDrop(t *testing.T) {
 	partitiontest.PartitionTest(t)
+	t.Parallel()
```

Comment:
This test checks the global variable `transactionMessagesDroppedFromPool `
One day, it may fail when another test changes its value. 
And this is a super fast test 160ms.